lib/ansible/playbook/play.py

```diff
@@ -104,6 +104,18 @@ def __init__(self):
     def __repr__(self):
         return self.get_name()
 
+    @property
+    def ROLE_CACHE(self):
```

Comment:
Should we deprecate this?

Reply:
I was thinking of doing so, but was not going to do it in this release.  I'll think on it.